manual/manual/cmds/intf-c.etex

```diff
@@ -1557,22 +1560,25 @@ using {\tt Is_exception_result} (followed by {\tt Extract_exception} if
 appropriate).
 
 The "-output-obj" option can also be used to obtain the C source file.
-More interestingly, the same option can also produce directly a shared
-library (".so" file, ".dll" under Windows) that contains the OCaml
-code, the OCaml runtime system and any other static C code given to
-"ocamlc" (".o", ".a", respectively, ".obj", ".lib"). This use of
-"-output-obj" is very similar to a normal linking step, but instead of
-producing a main program that automatically runs the OCaml code, it
-produces a shared library that can run the OCaml code on demand. The
-three possible behaviors of "-output-obj" are selected according
-to the extension of the resulting file (given with "-o").
+More interestingly, the "-output-complete-obj" option can also produce
+directly a shared library (".so" file, ".dll" under Windows) that
+contains the OCaml code, the OCaml runtime system and any other static
+C code given to "ocamlc" (".o", ".a", respectively, ".obj", ".lib").
+This use of "-output-complete-obj" is very similar to a normal linking
+step, but instead of producing a main program that automatically runs
+the OCaml code, it produces a shared library that can run the OCaml
+code on demand. The three possible behaviors of "-output-complete-obj"
```

Comment:
Maybe I don't understand something but isn't it the case that: 

1. `-output-obj`: `.{o,a}` -> only the ocaml program, `.so` -> ocaml program + runtime libraries + autolink (?) 
2. `-output-complete-obj`: `.{o,a,so}` -> ocaml program + runtime libraries + autolink

So I'm not sure I understand this "three possible behaviours" of `-output-complete-obj`.

Reply:
As far as I understand,  the two options  `-output-obj` and `-output-complete-obj` have the same behavior when producing a shared library or a c file. Would it not make sense to mention both of them in the previous paragraph?